### PR TITLE
Update project with calculator layout

### PR DESCRIPTION
**Pull request by CarlosE-Feliz**
# Math Magicians: Layout  🎉 

> Set up the layout to develop a React application💪🏽 

Additional description of the project and its features. ➕ 

## Technical set up
- Set up the repository on GitHub and use Gitflow.
- Delete all the boilerplate from CRA (text, images, styles).
- Create a directory called components.
- Inside components, create a new Calculator.js file.
- In Calculator.js, create a React component that matches this design:
image

- Don't add any extra design.
- Develop just the UI of the calculator; the math logic to make it work will be added in the next project.
- Use class based components.
- Once the Calculator is ready and imported it to main component (App.js).
- Tested  in the browser; it render without problems.

## Built With ❤️ 
- CSS
- JavaScript
- HTML
- React

### NOTE TO REVIEWER 😉 ❤️  
-

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    <tr className="calComp">9</tr>
+    <tr className="opeComp">*</tr>
+    <tr className="calComp">4</tr>
+    <tr className="calComp">5</tr>
+    <tr className="calComp">6</tr>
+    <tr className="opeComp">-</tr>
+    <tr className="calComp">1</tr>
+    <tr className="calComp">2</tr>
+    <tr className="calComp">3</tr>
+    <tr className="opeComp">+</tr>
```
Comment (Dipeshtwis) [inline src/components/Calculator.js]: - [x] There are duplicate `+` signs there, instead, it should be the `÷` (division) sign in one of the place.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+);
+
+function Calculator() {
+  return (
+    <div id="main" className="main">
+      <ShowText />
+      <DivOperations />
+    </div>
+  );
+}
```
Comment (Dipeshtwis) [inline src/components/Calculator.js]: - [x] The `Calculator.js` component should be class-based. In this milestone, the objective is to practice the class-based React component. 

- [x] Also, make a separate file for another component and import it into the file where you want to use that component. Here, you are using the `ShowText` and `DivOperation` components in the same file which is not good and will make the code structure clumsy.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @xJokeer, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Linters are passing :white_check_mark: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

- [x] You have removed almost all the file which was not required, but still, some of them are left, Please Remove `App.test.js`, `reportWebVitals.js` and `setupTests.js`, as we are not testing anything for this moment.

**_Also check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (CarlosE-Feliz) [reply src/components/Calculator.js]: Thanks! I had not realized that 🤣

File: src/components/Calculator.js (hunk 2 above)
Comment (CarlosE-Feliz) [reply src/components/Calculator.js]: ohh I see, I'll implement that!

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @xJokeer, 

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**